### PR TITLE
Responses to Highlights and Questions

### PR DESCRIPTION
**Pull request by Deva0770**

### CODE REVIEW

File: QUESTIONS.md [hunk 1]
```diff
@@ -3,3 +3,10 @@
 What questions can you come up with?
 
 ## Cohort Questions
+1 Need more help on exercies on writing code to connect to the front end.
+2 Some challenges to create a code to connect to the databse and insert a row or create a column in a database.
+3 Need advise on how to maintain cotinuity of thought while working on an particular project and keeping in mind the concepts and how they are all connected.
+4 How do we all stay connected after this training batch is over.
```
Comment (jessicareynolds) [inline QUESTIONS.md:20]: +1 I agree! Good question

File: QUESTIONS.md [hunk 2]
```diff
@@ -3,3 +3,10 @@
...
+5 Who would be available after this 18 weeks is over in case we have some technical questions.
```
Comment (jessicareynolds) [inline QUESTIONS.md:22]: +1

Comment (hollisjamison) [inline QUESTIONS.md:22]: Me for sure if you can't ever reach a TA.

File: HIGHLIGHTS.md [hunk 3]
```diff
@@ -7,3 +7,7 @@ What highlights can you come up with to share with the class?
 What do you really like? What can you find for fun facts related to any of these topics online?
 
 ## Cohort Highlights
+
```
Comment (alphaComara) [inline HIGHLIGHTS.md:10]: LGTM Deva really

File: HIGHLIGHTS.md [hunk 4]
```diff
@@ -7,3 +7,7 @@ What highlights can you come up with to share with the class?
...
+Weekly challenges have been very helpful as practice.
```
Comment (alphaComara) [inline HIGHLIGHTS.md:11]: 100% on board with this very helpful

File: QUESTIONS.md (hunk 2 above)
Comment (Deva0770) [reply QUESTIONS.md:22]: Nice of you Hollis

**Pull request merged**